### PR TITLE
feat: Add welcome carousel if user is signing in for the first time

### PR DESCRIPTION
**Pull request by tinaszheng**
image
image
image

### CODE REVIEW

File: src/components/common/Account.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+"use client"
```
Comment (tinaszheng) [inline src/components/common/Account.tsx:1]: moved account stuff to its own file

File: src/app/welcome/page.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      router.push("/dashboard")
+    }
+  }
+
+  const isLastCard = carouselApi && !carouselApi.canScrollNext()
+
+  return (
+    <div className="flex flex-col items-center pt-20 pb-4 bg-backgroundSecondary min-h-lvh">
+      <div className="bg-background w-2/3 p-16 text-center rounded-3xl items-center flex flex-col gap-6">
+        <div className="py-0.5 px-2 bg-backgroundSecondary rounded-full">
```
Comment (bswags) [inline src/app/welcome/page.tsx:33]: i made a lil `Badge` component that I updated in #9 that you can grab for this for #reusability

File: tailwind.config.ts [hunk 3]
```diff
@@ -67,6 +67,7 @@ const config = {
           DEFAULT: "hsl(var(--card))",
           foreground: "hsl(var(--card-foreground))",
         },
+        optimismRed: "#FF0420",
```
Comment (bswags) [inline tailwind.config.ts:70]: this is weirdly in there as the "destructive" color rn if you wanna use that in your styling

File: src/app/welcome/page.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+        <Carousel setApi={setCarouselApi}>
+          <CarouselContent>
+            <WelcomeCard card="intro" />
+            <WelcomeCard card="projects" />
+            <WelcomeCard card="funding" />
+          </CarouselContent>
+        </Carousel>
+        <Dots total={3} current={currIndex} />
+        <button
```
Comment (bswags) [inline src/app/welcome/page.tsx:44]: annoying nit but the `Button` in the shadcn components/ui dir should be styled for this (can grab similar variant/etc. from my PR)

**Review by bswags — APPROVED** [review]

File: src/app/welcome/page.tsx (hunk 4 above)
Comment (tinaszheng) [reply src/app/welcome/page.tsx:44]: Oh so true bestie

File: tailwind.config.ts (hunk 3 above)
Comment (tinaszheng) [reply tailwind.config.ts:70]: Oh weird ok I will rename that variable hahaha

File: src/app/welcome/page.tsx (hunk 2 above)
Comment (tinaszheng) [reply src/app/welcome/page.tsx:33]: Oh perfecttttt

**Pull request merged**